docs/android/deploy-test/debugging/index.md

```diff
@@ -44,5 +44,5 @@ describes how to use **logcat**.
 
 > [!WARNING]
 > Note that the **Xamarin Android Player** has been deprecated. For more information, see the [announcement in this
- blog post](https://blog.xamarin.com/live-from-dotnetconf-cycle-7-xamarin-studio-6-and-more/). In addition, the
-**Visual Studio Android Emulator** has been deprecated as of Visual Studio 2017.
+> blog post](https://blog.xamarin.com/live-from-dotnetconf-cycle-7-xamarin-studio-6-and-more/). In addition, the
```

Comment:
This inserts the symbol in the middle of link

Reply:
Good catch, i'll fold the link back to the previous line

Reply:
OK, I joined the link back with the previous line